md/release-notes.md

```diff
@@ -1,30 +1,86 @@
+
+Performances on realistic pages show a 20% improvement with an Ethernet connection, and a 40% improvement on 3G.
 
+#### Performance at following displays
+1. Pages and forms resources are now saved in the brwoser cache.
+Cache has been set to 6 months, but this is configurable.
 
-### Industrialization of Living Application (in Studio)
+2. Cache-busting has been implemented 
+As a consequence, any udpate in application artefacts is automatically visible by end users
+Attention: if you create or edit artifacts outside of the UI Designer, you will need to udpate the artifacts URL to trigger the cache-busting mechanism to reflect the updates.
 
-## Performance improvements
+Performances on realistic pages show a 33% improvement with an Ethernet connection, and a 86% improvement on 3G.
 
-### Engine startup and BPM Services resume
+### Engine performance improvement: startup and BPM Services resume
 
-A major improvement has been integrated in Bonita 7.6.0, especially for tenants with a lot of processes, processes with large dependencies or processes with a
-lot of dependencies
+A major improvement has been integrated, especially for tenants with a lot of processes, processes with large dependencies or processes with a lot of dependencies.
 
-Tests conducted both internally at Bonitasoft and on customer sites with real production data have shown the following decreases
+Tests conducted both internally at Bonitasoft and on customer's sites with real production data have shown the following decreases
 * execution time by a factor of 5 to 10
 * JVM Heap memory consumption by a factor of 2 to 3
 
-### Living Application
+### Workspace import moved to a menu in Bonita Studio
+Importing an existing workspace into a new version of the studio is no more available in the installer: it has been moved to a new menu "File > Import > Workspace..." in the studio, and is automatically prompted on the first start of the studio.
 
-To be documented
+### Button widget and forms integration improvements
+The integration of the forms/pages in Bonita (mobile) Portal or applications and the behavior of the Button widget have been improved in order to:
+ - avoid redirecting the current window when the form submitted is displayed in the (mobile) portal and in the applications (issues [BS-16774](https://bonitasoft.atlassian.net/browse/BS-16774) and [BS-16790](https://bonitasoft.atlassian.net/browse/BS-16790) )
+ - ignore the target URL on success when the form submitted is displayed in the (mobile) portal (issue [BS-16685](https://bonitasoft.atlassian.net/browse/BS-16685) ).
+In addition to those improvements, an embedded help as been added to the Button widget in the UI Designer, explaining this behavior.
+
+### User management page in Administrator Portal is all new 
+To insure the move of all Bonita Portal pages in AngularJS, the User "more" page in the Administrator Portal has been redesigned and reimplemented.
```

Comment:
Why not simply call the page "User details page"?
"more" has no functional meaning, in my view :sob:

Reply:
Because the button is called "More".